### PR TITLE
Implement the super admin for publisher app using django groups.

### PR DESCRIPTION
**Pull request by awais786**
Implement the global admin user for publisher app.

### CODE REVIEW

**Comment by awais786** [comment]
@ahsan-ul-haq @tasawernawaz @waheedahmed Please review.

File: course_discovery/apps/publisher/migrations/0013_create_admin_group.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+ADMIN_GROUP_NAME = 'Publisher Admins'
+
+
+def create_admin_group(apps, schema_editor):
+    Group = apps.get_model('auth', 'Group')
+    Group.objects.get_or_create(name=ADMIN_GROUP_NAME)
```
Comment (awais786) [inline course_discovery/apps/publisher/migrations/0013_create_admin_group.py]: I did minor change here using `get_or_create` instead of `create`.

**Comment by awais786** [comment]
This code is provided by @clintonb [Example of using a Django group](https://github.com/edx/course-discovery/commit/6ef5118c5faba4e2b9dc3c7d693e8915c705bb71).

File: course_discovery/apps/publisher/migrations/0014_create_admin_group.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+ADMIN_GROUP_NAME = 'Publisher Admins'
```
Comment (ahsan-ul-haq) [inline course_discovery/apps/publisher/migrations/0014_create_admin_group.py]: why not using constant value declared in course_discovery/apps/publisher/constants.py

File: course_discovery/apps/publisher/utils.py [hunk 3]
```diff
@@ -14,3 +15,13 @@ def is_email_notification_enabled(user):
         return user.attributes.enable_email_notification
 
     return True
+
+
+def is_global_admin(user):
```
Comment (ahsan-ul-haq) [inline course_discovery/apps/publisher/utils.py]: I think it would be better to change it to is_publisher_admin.

**Review by ahsan-ul-haq — CHANGES_REQUESTED** [review]

**Review by ahsan-ul-haq — APPROVED** [review]

**Comment by awais786** [comment]
@edx/ecommerce Please review this PR.

**Comment by awais786** [comment]
@bderusha would you please review this PR.

**Review by clintonb — APPROVED** [review]
Nit: when modifying others' commits, use fixup (http://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html) to retain the original authorship.

Remember to squash commits (which is also easily done using fixup).

**Comment by awais786** [comment]
I will update the PR tomorrow to maintain the authorship.

**Pull request merged**